src/cryptoadvance/specter/server_endpoints/settings.py

```diff
@@ -93,6 +93,12 @@ def bitcoin_core():
                 autodetect=autodetect,
                 datadir=datadir,
             )
+
+            if "tests" in test:
+                if not test["tests"]["connectable"]:
```

Comment:
Why test only connectable? I think it'd make sense to test all... Or maybe have an error message for each stage.

Reply:
Good catch! Fixed the issue by keeping the user notification binary, but while considering all the sub tests. Fixed this with the latest commit.

https://github.com/cryptoadvance/specter-desktop/pull/912/files#diff-3a59cc1aa05ae505db526d6d6861af0634ffee79409099c09b97d97a95cbfdeeR99